components/butterfly/src/server/mod.rs

```diff
@@ -828,14 +828,14 @@ impl Server {
     }
 
     /// Get all the Member ID's who are present in a given service group, and count towards quorum.
-    pub fn get_total_population(&self, key: &str) -> usize {
-        let mut total_pop = 0;
+    fn get_total_population(&self, key: &str) -> Vec<String> {
+        let mut total_pop = vec![];
```

Comment:
Could we pre-allocate a vector using `.with_capacity(...)` here? We could end up saving some memory with large groups.

Reply:
Yeah, that's a good idea. I'll take a look at doing that in a separate PR.